### PR TITLE
✨ Update template overview page including SSR

### PR DESCRIPTION
**Pull request by robinvanopstal**

### CODE REVIEW

File: frontend/icons/download-dark.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<symbol id="download-dark" viewBox="0 0 25 25">
```
Comment (matthiasrohmer) [inline frontend/icons/download-dark.svg]: Have the icons been piped through SVGOMG?

File: frontend/scss/components/organisms/format-filter.scss [hunk 2]
```diff
@@ -0,0 +1,132 @@
+@import '../../extends';
```
Comment (matthiasrohmer) [inline frontend/scss/components/organisms/format-filter.scss]: Please use absolute imports here and sort them as in the other files 😉

File: frontend/scss/components/organisms/format-filter.scss [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  background: color('white');
+  cursor: pointer;
+  box-shadow: 0 5px 25px 0px rgba(0,0,0,0.25);
+  transition: 200ms;
+
+  &:hover {
+    box-shadow: 0 15px 35px 0px rgba(0,0,0,0.15);
+  }
+
+  &.websites .#{atom('ico')} { fill: #CCC; font-size: 21px; }
```
Comment (matthiasrohmer) [inline frontend/scss/components/organisms/format-filter.scss]: Isn't `#CCC` a proper color in _variables.scss?

File: frontend/templates/views/partials/format-filter.j2 [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{% do doc.styles.addCssFile('css/components/organisms/format-filter.css') %}
```
Comment (matthiasrohmer) [inline frontend/templates/views/partials/format-filter.j2]: Hm... Can we find a better name for this? For me format-filter is confusing with the filter dropdown we have on the documentation pages

File: frontend/icons/download-dark.svg (hunk 1 above)
Comment (robinvanopstal) [reply frontend/icons/download-dark.svg]: Yep :)

File: frontend/templates/views/partials/format-filter.j2 (hunk 4 above)
Comment (robinvanopstal) [reply frontend/templates/views/partials/format-filter.j2]: I was thinking about naming it `format-filter-bubble-list` but that seemed a little too long for me? 😄

Comment (matthiasrohmer) [reply frontend/templates/views/partials/format-filter.j2]: What about `format-filter-bubbles`? 😄At least that's a little shorter.

Comment (robinvanopstal) [reply frontend/templates/views/partials/format-filter.j2]: Okay, done 📟

**Comment by sebastianbenz** [comment]
What's the status of this?

**Comment by matthiasrohmer** [comment]
@robinvanopstal, this would now be ready to get merged as SSR is fully working now. I don't have enough context to resolve the merge conflicts though. Would be awesome if you could have a look with @sebil maybe.

**Comment by sebil** [comment]
@googlebot I fixed it.

**Comment by robinvanopstal** [comment]
@googlebot I fixed it.

**Comment by sebastianbenz** [comment]
Closing this for now as template pages get reworked anyway.

**Pull request closed without merging**